### PR TITLE
Cnft2 1469 manage tabs drag drop reorder

### PR DESCRIPTION
**Pull request by jlee-enquizit**
## Description

Here we implement the drag drop reorder functionality to ManageTabs.
The DragDropContext was refactored some to accomodate Tabs but also improve functionality in the Section Reorder Modal.
Some ManageTabs design updates were added to this branch.

## Tickets

* https://cdc-nbs.atlassian.net/browse/CNFT2-1469

## Steps to verify

1. Select a Page from the Page Library
2. Click the `Manage Tabs` button in the Tabs navbar.
3. Drag a tab by the handle and move into a new position.
4. Tab order should update, as well as the Tab order on the Page

### CODE REVIEW

**Review by achapm — CHANGES_REQUESTED** [review]
Please resolve merge conflicts

File: apps/modernization-ui/src/styles/_uswds.scss [hunk 1]
```diff
@@ -62,8 +62,10 @@ button.usa-button {
     svg {
         vertical-align: bottom;
     }
+    // display: flex;
```
Comment (achapm) [inline apps/modernization-ui/src/styles/_uswds.scss:65]: Unless you see a good reason to keep this, we should remove the comments here and the line below. Its just adding cruft.

**Review by achapm — APPROVED** [review]
Please see the comments below, otherwise 👍

File: apps/modernization-ui/src/apps/page-builder/pages/EditPage/ReorderSubsection/ReorderSubsection.tsx [hunk 2]
```diff
@@ -47,29 +49,29 @@ export const ReorderSubsection = ({ subsection, index, visible }: Props) => {
...
-                                      );
-                                  })
-                                : null}
-                            {prov.placeholder}
-                        </div>
-                    )}
-                </Droppable>
-            </div>
-        </div>
+                    <div className={`reorder-subsection__questions ${!questionsOpen ? 'closed' : ''}`}>
```
[the remark this answers is not in the document]
Comment (jlee-enquizit) [reply apps/modernization-ui/src/apps/page-builder/pages/EditPage/ReorderSubsection/ReorderSubsection.tsx:52]: looks okay here

**Review by mpeels — APPROVED** [review]

**Pull request merged**